### ISSUE TITLE
Option to `alias` to File System

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy-fetch#54]

**Comment by ryantownsend** [comment]
@zachleat came here to guage interest in writing the fetched responses to a static output location for use in edge functions (rather than just during build)... the title of this issue sounds like it _might_ be the same thing but I don't really have the context from the linked code. Is that what you're thinking here or shall I open a separate issue?

**Comment by zachleat** [comment]
@ryantownsend if I understand you correctly, this already happens via the `.cache` folder (by default) but you can change this to whatever you’d like here: https://www.11ty.dev/docs/plugins/fetch/#cache-directory

**Comment by ryantownsend** [comment]
@zachleat is there a way to configure the actual filename too, without a fingerprint so it can be statically imported into a JavaScript function?

**Comment by zachleat** [comment]
@ryantownsend yeah, 100%: https://www.11ty.dev/docs/plugins/fetch/#change-the-cache-filename

**Comment by ryantownsend** [comment]
@zachleat amazing, not sure how I missed that! Well, guess there's no reason for me to be opening any PRs now 😄
